### DISCUSSION TITLE
QUarto latex template does not include affiliation or institution?

### DISCUSSION THREAD

**Discussion by statzhero** [quarto-dev/quarto-cli#10642] [Q&A]
### Bug description

Not sure this is a bug at all, however: 

The documentation talks at length about [authors and affiliations](https://quarto.org/docs/journals/authors.html) but then the default [latex template](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/pdf/pandoc/title.tex) does not render an affiliation: is the idea that quarto users should edit the `title.tex` themselves? 

### Your environment

``` bash
➜  ~ quarto --version
1.3.450
```

``` r
R version 4.3.2 (2023-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Sonoma 14.4.1
```

### Quarto check output

``` bash
➜  ~ quarto check 

[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.450
      Path: /Users/rico/Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.12.5
      Path: /opt/homebrew/opt/python@3.12/bin/python3.12
      Jupyter: 5.7.2
      Kernels: python3

[✓] Checking Jupyter engine render....OK
```

**Comment by mcanouil** [comment]
Unfortunately, we can't use your report.
The sections you removed about providing reproducible example were needed.
Additionally, when not sure if you found a bug, prefer opening a GitHub Discussion rather than an Issue.

Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by cscheid** [comment]
I'm going to convert to a Q&A discussion.

@statzhero Yes, that's the idea, eg in conjunction with the templates we offer in https://github.com/quarto-journals/
